pkg/client/leaderelection/resourcelock/endpointslock.go

```diff
@@ -23,14 +23,14 @@ import (
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/kubernetes/pkg/api/v1"
-	"k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
+	corev1client "k8s.io/kubernetes/pkg/client/clientset_generated/clientset/typed/core/v1"
```

Comment:
might be out of scope, but why don't we use client-go here?
Would that require a big "switch-all" to client-go?